ui/apps/ui/src/app/components/search-input/search-input.component.ts

```diff
@@ -89,7 +89,7 @@ export class SearchInputComponent implements OnInit {
   );
 
   collectionFcAdvForm = new FormControl<ICollectionTagsConfig>(
-    this.collectionFcAdv[0],
+    this.collectionFcAdv[2],
```

Comment:
What exactly does it fix? 
init value from  `{ name: 'Author' },`to  `{ name: 'In title' }`
Later on I see: `this.collectionFcAdvForm.reset(this.collectionFcAdv[2]);` 
The init value was the issue why searching on page 6 wasn't working?
Maybe place some comment about it in the code. Its kinda a magical number

Reply:
itsa. long story, but its not related to search, its related to non existing provider search forms, i choosen the one that exist on both sides